piecemeal-plugin/src/main/kotlin/dev/bnorm/piecemeal/plugin/fir/factory.kt

```diff
@@ -184,15 +181,15 @@ fun FirExtension.createPropertyBuilderValue(
   return property
 }
 
-@OptIn(ExperimentalTopLevelDeclarationsGenerationApi::class)
 fun FirExtension.createFunPiecemealDsl(
   companionClassSymbol: FirClassSymbol<*>,
   callableId: CallableId,
 ): FirSimpleFunction? {
   val piecemealClassId = companionClassSymbol.classId.outerClassId!!
   val piecemealClassSymbol = session.findClassSymbol(piecemealClassId)!!
-  val builderClassSymbol = session.findClassSymbol(piecemealClassId.builder) ?: return null
-  val builderType = fun1Ext(session, receiver = builderClassSymbol)
+  val mutableClassSymbol = session.findClassSymbol(piecemealClassId.mutable) ?: return null
+  val builderType = fun1Ext(session, receiver = mutableClassSymbol)
+  val paramName = "builder"
```

Comment:
I figured this is still an appropriate name for the `build { }` function's lambda parameter.

Reply:
The stdlib calls it `builderAction`. I don't think it matters all that much either way.